chain/client/src/info.rs.orig

```diff
@@ -0,0 +1,291 @@
+use std::cmp::min;
```

Comment:
I think this is an artifact of some merge conflict, so we need to remove it.

Reply:
removed